doc/05-Authentication.md

```diff
@@ -171,9 +171,9 @@ Option                   | Description
 -------------------------|-----------------------------------------------
 backend                  | **Required.** Specifies the backend type. Can be set to `ldap`, `msldap`.
 resource                 | **Required.** The name of the LDAP resource defined in [resources.ini](04-Resources.md#resources).
-user\_class              | **Optional.** LDAP user class. Defaults to `user`.
-user\_name\_attribute    | **Optional.** LDAP attribute which contains the username. Defaults to `sAMAccountName` with `msldap` and `uid` with `ldap`.
+base_dn                  | **Optional.** LDAP base dn for groups.
```

Comment:
`user_class` and `user_name_attribute` are valid configuration options here and shouldn't be removed. Though, `user_class` is not quite correct. The description is missing the `msldap` default and the `base_dn` may also include a better description:

```suggestion
user\_class              | **Optional.** LDAP user class. Defaults to `inetOrgPerson` with `msldap` and `user` with `ldap`.
user\_name\_attribute    | **Optional.** LDAP attribute which contains the username. Defaults to `sAMAccountName` with `msldap` and `uid` with `ldap`.
base_dn                  | **Optional.** LDAP base dn for groups. Leave empty to select all groups available using the specified resource.
```